src/SDK/Client/ClientApps.spec.ts

```diff
@@ -0,0 +1,86 @@
+import Identifier from "@dashevo/dpp/lib/Identifier";
+import {expect} from 'chai';
+import {ClientApps} from "./ClientApps";
+import 'mocha';
+import exp from "constants";
+
+describe('ClientApps', () => {
+    let apps;
+    it('constructor', function () {
+        apps = new ClientApps();
+        expect(apps.apps).to.deep.equal({});
+
+        const appsFromProps = new ClientApps([{
+            contractId: '3VvS19qomuGSbEYWbTsRzeuRgawU3yK4fPMzLrbV62u8',
+            contract: null,
+            aliases: ['dpns']
```

Comment:
It's just `alias: string` in code. Do we really need several names for data contract? Maybe we just rename it to `name` and have it only one?

Reply:
Well, the whole situation is happening when you do multiple name for the same data contract, or when you do another contract id for the same name. 
It's because I did not plan for this usage that we end up having a bug in console

Reply:
Hm.. Sorry, I don't understand why do we need several names for the same contract. Or name different contracts with the same name.

Reply:
Look at #150.